Src/Support/Google.Apis.Auth/OAuth2/ComputeCredential.cs

```diff
@@ -118,7 +118,8 @@ private static async Task<bool> IsRunningOnComputeEngineNoCache()
                     cts.CancelAfter(MetadataServerPingTimeoutInMilliseconds);
                     try
                     {
-                        var httpRequest = new HttpRequestMessage(HttpMethod.Get, MetadataServerUrl);
+                        var httpRequest = new HttpRequestMessage(HttpMethod.Get, MetadataServerUrl);

```

Comment:
I can't tell in GitHub what's changed on this line of code. Was there some whitespace that I'm failing to see?

Reply:
Turns out I use windows line endings 😄